### PR TITLE
[#175716267] Adding member to team

### PR DESCRIPTION
**Pull request by miljan-kolosek**
Tasks I didn't do: 
- display 'invited' (pending) users
- display remaining time

### CODE REVIEW

File: app/controllers/invitations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class InvitationsController  < Devise::InvitationsController
+
+	def invite_resource
+      User.invite!(invite_params, current_inviter) do |invitable|
+      invitable.update!(role: invite_params[:role])
```
Comment (tamara-kolosek) [inline app/controllers/invitations_controller.rb:5]: please correct identation

File: app/controllers/team_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class TeamController < ApplicationController
+
+	def index
+		@users = User.all
+	end
+
+	def new_user_invite
```
Comment (tamara-kolosek) [inline app/controllers/team_controller.rb:7]: Make policy for this controller so you can prevent non admin users to take these actions

File: app/views/team/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+	<h3><%= link_to 'Invite people', new_user_invite_path , remote: true , class: 'btn btn-primary' %></h3>
+
+	<div id="invite"></div>
+
+	<br /><br />
+
+	<h3> Pending users </h3>
+	<!-- <% @users.each do |pending_user| %>
+		<% if pending_user.invitation_accepted_at != nil %>
```
Comment (tamara-kolosek) [inline app/views/team/index.html.erb:12]: do the separation of users by adding appropriate scope in User model class, so you wont have these many checks in view

**Review by tamara-kolosek — CHANGES_REQUESTED** [review]

File: app/controllers/invitations_controller.rb (hunk 1 above)
Comment (miljan-kolosek) [reply app/controllers/invitations_controller.rb:5]: Identation in my editor is fine, github preview is different than mine in editor.

File: app/controllers/team_controller.rb [hunk 4]
```diff
@@ -1,7 +1,11 @@
 class TeamController < ApplicationController
 
+	before_action :authenticate_user!
+
 	def index
 		@users = User.all
+		@pending = User.where.not(invitation_token: nil)
```
Comment (tamara-kolosek) [inline app/controllers/team_controller.rb:7]: use rails scopes https://devblast.com/b/jutsu-11-rails-scopes

and then you will be able to call it direct from view

**Review by tamara-kolosek — APPROVED** [review]

**Pull request merged**